<?xml version="1.0" encoding="UTF-8"?>
<project version="4">
  <component name="DataSourceManagerImpl" format="xml" multifile-model="true">
    <data-source source="LOCAL" name="Django default" uuid="063315fc-a013-444e-9d39-68b15db4b19f">
      <driver-ref>mysql.8</driver-ref>
      <synchronize>true</synchronize>
      <imported>true</imported>
      <remarks>$PROJECT_DIR$/kpi/settings.py</remarks>
      <jdbc-driver>com.mysql.cj.jdbc.Driver</jdbc-driver>
      <jdbc-url>jdbc:mysql://localhost:3306/kpi</jdbc-url>
    </data-source>
    <data-source source="LOCAL" name="@47.94.148.26" uuid="6029cbf6-34b2-4cd6-ade6-6c295285ed70">
      <driver-ref>mysql.8</driver-ref>
      <synchronize>true</synchronize>
      <jdbc-driver>com.mysql.cj.jdbc.Driver</jdbc-driver>
      <jdbc-url>jdbc:mysql://47.94.148.26:3306</jdbc-url>
    </data-source>
    <data-source source="LOCAL" name="MySQL - @localhost" uuid="ef642735-94bd-4865-a460-7f253a5feb89">
      <driver-ref>mysql.8</driver-ref>
      <synchronize>true</synchronize>
      <jdbc-driver>com.mysql.cj.jdbc.Driver</jdbc-driver>
      <jdbc-url>jdbc:mysql://localhost:3306</jdbc-url>
    </data-source>
  </component>
</project>